### ISSUE TITLE
Won't let me save anymore

### ISSUE THREAD

**Issue by sarahsaidso** [photopea/photopea#1910]
My chromebook just had an update, and I think that somehow made it so I can't save anything from photopea anymore. Any way I try to export it, a browser window pops up with the url "about:blank#blocked". When I try to save as a PSD, it won't save as "PSD file", but just "File", which my chromebook is then not able to open. Ad block is disabled and cookies are enabled. I've been using photopea for months and have never had an issue with saving until now, and the only variable being the update. I don't know if there's some simple solution to this that's obvious, but I just can't figure it out.

**Comment by photopea** [comment]
Hi, are you using a standard release of Chrome (and not Chrome Beta, Canary, etc)?

Can you try Photopea in Incognito mode? If it works there, then, some extension is blocking Photopea from working correctly.

**Comment by sarahsaidso** [comment]
Standard release.

Incognito mode worked! I removed the ad block extension in case that was causing it, but it didn't change. I'll remove some more and see how that goes.

**Comment by sarahsaidso** [comment]
Ok, so it looks like the new chrome os update made it so that having ANY extension will prevent photopea from saving properly. That's frustrating.

**Comment by photopea** [comment]
Are you sure? How many extensions have you tried? Every extension alone blocks saving files?

**Comment by sarahsaidso** [comment]
I one by one I toggled the "work in incognito" on the extension, and they all prevented it from working. I also turned all but one off in normal chrome, and the problem kept happening. So I just removed them all. It didn't necessarily stop it from saving, it would save PDF files as just "Files", and when I would click to open that, it would pop up another window to download that file again, so it was impossible to save it in the right format, and it was impossible to even open it. 

After removing them all I have since restarted my chromebook, and I added the dark theme extension, and I was able to export the file to pdf no problem. I added the pdf merger, and that is the culprit. I changed it settings to only be active when I click it instead of working on all sites, and that solved it. 

Thank you for your help, I would never had thought to check if an extension that has always been there would have been the problem .
